frontend/src/features/leases/detail/LeaseRouter.tsx

```diff
@@ -14,7 +14,7 @@ export const LeaseRouter: React.FunctionComponent<ILeaseRouterProps> = React.mem
       {Array.from(leasePages.entries()).map(([pageName, page]) => (
         <AppRoute
           path={`${path}/${pageName}`}
-          component={page.component ?? React.Fragment}
+          customComponent={page.component}
```

Comment:
different change here then what you did for the ContactRouter. It seems like passing null/undefined for customcomponent isn't valid so perhaps this is the correct version of this change?

Reply:
I think the React.Fragment part was just being used in case the page.component was not returning a valid component. My guess is that we used that during development when some pages were not returning a valid component. Now all pages should return a valid one, so that extra code is no longer needed,